README.md

```diff
@@ -11,7 +11,7 @@
 
 
 
-Cloud Document API client for Node.js
+Documentai client for Node.js
```

Comment:
That don't look right - I can't remember what populates this though 😆

Reply:
Changed.

Reply:
I think that it's populated by the `description` field in the package.json file